jujugui/static/gui/src/app/assets/css/_components.scss

```diff
@@ -6,6 +6,7 @@
 @import "../../components/boolean-config/boolean-config";
 @import "../../components/budget-chart/budget-chart";
 @import "../../components/budget-table/budget-table";
+@import "../../components/budget-table/row/row";
```

Comment:
Does this need to be nested in a row directory? I wonder if `/budget-table/_row.scss` would be neater?